pandas/core/groupby/groupby.py

```diff
@@ -5829,6 +5808,66 @@ def func(df):
                     stacklevel=find_stack_level(),
                 )
 
+        if axis == 1:
+            try:
+                if self.obj.ndim == 1:
```

Comment:
can we have self.obj.ndim == 1 with axis==1?

Reply:
Ah, thanks. This has been removed.